results/intfloat__multilingual-e5-small/WikiClusteringFastP2P.json

```diff
@@ -0,0 +1,25 @@
+{
+  "dataset_revision": "d4d92f8f28be71035be6a96bdfd4e200cf62faa8",
+  "mteb_dataset_name": "WikiClusteringFastP2P",
+  "mteb_version": "1.8.3",
+  "test": {
+    "bs": {
```

Comment:
Hmm somehow only 1 subset is reported.

Reply:
@x-tabdeveloping this is likely the first multi-lingual task that AbsTaskClusteringFast has encountered. Maybe `_evaluate_monolingual` should be called in the `for lang in self.dataset` loop instead of outside of it.

Reply:
Found a fix. Will push in next commit.